elasticms-cli/src/Client/File/FileReaderImportConfig.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\CLI\Client\File;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class FileReaderImportConfig
```

Comment:
To me this config applies to CSV files:
```suggestion
class FileReaderImportCsvConfig
```

Reply:
No its file import config. Ouuid expression, deleteExisting, generate hash is all about import not csv